### DISCUSSION TITLE
Hide Last Edited By.

### DISCUSSION THREAD

**Discussion by mdeepakplmnew** [requarks/wiki#4772] [Help / Questions]
Is there any way to Hide the Last Edited by Column ..

![image](https://user-images.githubusercontent.com/93391152/145551345-c618a2c7-6dd2-43cb-bd00-c7a45e75cde8.png)

**Comment by mdeepakplmnew** [comment]
anyone know about this ?

**Reply by saimen-ti** [reply]
i would also like to know if it's possible to hide?

**Comment by fnafox** [comment]
You can do this with custom css, I recommend checking out this repo as it has this and other functionality and has instructions: https://github.com/madodig/wikijs-customization

**Reply by resherry2020** [reply]
Hi, I installed it but it doesn't work as it said... Did you face the same issue as me?

**Reply by fnafox** [reply]
It works ok for me! Double check if you've followed the instructions.

make sure you have toggled the options in the theme section under Head HTML Injection

also you must have the scripts in the Body Injection and they need to be accessible!
